homeassistant/components/simplefin/config_flow.py

```diff
@@ -0,0 +1,73 @@
+"""Config flow for SimpleFIN integration."""
+
+from typing import Any
+
+from simplefin4py import SimpleFin
+from simplefin4py.exceptions import (
+    SimpleFinAuthError,
+    SimpleFinClaimError,
+    SimpleFinInvalidAccountURLError,
+    SimpleFinInvalidClaimTokenError,
+    SimpleFinPaymentRequiredError,
+)
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
+
+from .const import CONF_ACCESS_URL, DOMAIN, LOGGER
+
+
+class SimpleFinConfigFlow(ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for the initial setup of a SimpleFIN integration."""
+
+    async def async_step_user(
```

Comment:
Let's abort if we already can find the access_url in the exisitng enntries

Reply:
So something like this?
```python
access_url = user_input.get(CONF_ACCESS_URL, None)
if not access_url:
    return self.async_abort(reason="url_error")
```

I'm not sure I see the flow but ok :)

(If the user gets the form and puts nothing in the box won't the access URL be a blank string - not `None`?)

Reply:
@joostlek  - Can you resolve this one if you are happy with it.

Reply:
no, self._async_abort_entries_match